chainer/functions/__init__.py

```diff
@@ -99,6 +99,7 @@
 from chainer.functions.pooling import roi_pooling_2d
 from chainer.functions.pooling import spatial_pyramid_pooling_2d
 from chainer.functions.pooling import unpooling_2d
+from chainer.functions.subpixel import depth2space as depth2space_
```

Comment:
`_` suffix is unnecessary. https://github.com/tjtorres/chainer/blob/d665ea8c7047b63a858c26a690814194545c7ab6/chainer/functions/__init__.py#L121